reddit/base.py

```diff
@@ -2,18 +2,14 @@
 from dotenv import load_dotenv, find_dotenv
 load_dotenv(find_dotenv())
```

Comment:
Once the environment variables have been loaded in the config module, they won't need to be loaded again.